app/src/main/java/com/practicum/playlistmaker/presentation/ui/searchActivity/SearchActivity.kt

```diff
@@ -147,8 +141,9 @@ class SearchActivity : AppCompatActivity() {
     override fun onDestroy() {
         super.onDestroy()
         handler.removeCallbacks(searchRunnable)
+        val currentRunnable = useCaseRunnable
+        currentRunnable?.let { handler.removeCallbacks(currentRunnable) }
```

Comment:
```suggestion
        useCaseRunnable?.let(handler::removeCallbacks)
```
тут можно сократить чуть чуть

Reply:
Спасибо! Поправил